src/Qmmands/ICommandService.cs

```diff
@@ -6,10 +6,55 @@
 namespace Qmmands
 {
     /// <summary>
-    ///     Provides a framework interface for creating text based commands.
+    ///     Represents extracted interface from <see cref="CommandService"/>.
```

Comment:
Represents **the** extracted interface **of** `<see cref="CommandService"/>` ?
Looks more English, you know.